### PR TITLE
Enable three scenarious for Scheduling APP

### PR DESCRIPTION
**Pull request by korneliakobiela**
[Issue] N/A
[Problem] While creating schedule APP. there was always scenario for
Turn Device on and off
[Solution] - Creating possibility for other scenarios

Signed-off-by: Kornelia Kobiela <k.kobiela@samsung.com>

### CODE REVIEW

File: examples/mobile/ScheduleApp/create-event-off.html [hunk 1]
```diff
@@ -87,7 +87,7 @@
 			<a class="ui-btn app-event-cancel" data-style="flat" href="#main">
 				Cancel
 			</a>
-			<a class="ui-btn app-event-done" data-style="flat" href="#">
+			<a class="ui-btn app-event-done" data-style="flat" href="#" data-type="off">
```
Comment (grzegorz-czajkowski) [inline examples/mobile/ScheduleApp/create-event-off.html:90]: data-* attributes are handled by TAU framework. I'd not mix them application data. How about just "type" atrribute?

File: examples/mobile/ScheduleApp/js/app.js [hunk 2]
```diff
@@ -185,7 +187,20 @@
 			</div>
 		</div>`
 
-			container.innerHTML += HTMLTemplate;
+			switch (type) {
+				case "on":
```
Comment (grzegorz-czajkowski) [inline examples/mobile/ScheduleApp/js/app.js:191]: Extend this case for "onoff" as well.
case "on":
case "onoff":

File: examples/mobile/ScheduleApp/js/app.js [hunk 3]
```diff
@@ -185,7 +187,20 @@
...
+					container.innerHTML += HTMLTemplateOn;
+					break;
+				case "off":
```
Comment (grzegorz-czajkowski) [inline examples/mobile/ScheduleApp/js/app.js:194]: case "off":
case "onoff":

File: examples/mobile/ScheduleApp/js/app.js [hunk 4]
```diff
@@ -185,7 +187,20 @@
...
+					container.innerHTML += HTMLTemplateOff;
+					break;
+				case "onoff":
```
Comment (grzegorz-czajkowski) [inline examples/mobile/ScheduleApp/js/app.js:197]: This can be removed then.

**Review by grzegorz-czajkowski — COMMENTED** [review]
Comments to consideration.

File: examples/mobile/ScheduleApp/create-event-off.html (hunk 1 above)
Comment (korneliakobiela) [reply examples/mobile/ScheduleApp/create-event-off.html:90]: Data attributes are common HTML attributes. Custom ones without a prefix are possible only for Custom Elements https://developer.mozilla.org/en-US/docs/Learn/HTML/Howto/Use_data_attributes

File: examples/mobile/ScheduleApp/js/app.js (hunk 3 above)
Comment (korneliakobiela) [reply examples/mobile/ScheduleApp/js/app.js:194]: Our eslint rules don't allow to duplicate case labels. This is defined in `eslint:recommended` which we follow in our ruleset.

**Review by grzegorz-czajkowski — APPROVED** [review]
LGTM.

**Pull request merged**